src/Resources/contao/library/Contao/DcaLoader.php

```diff
@@ -85,7 +85,10 @@ public function load($blnNoCache=false)
 					include $file;
 				}
 			}
-			catch (\InvalidArgumentException $e) {}
+			catch (\InvalidArgumentException $e)
+			{
+				\System::getContainer()->get('monolog.logger.contao')->notice($e->getMessage(), ['exception' => $e]);
```

Comment:
Well we can't change that for BC reasons, extending from `\InvalidArgumentException` is BC.